src/lib/puppeteer/screenshot/getInstagramPostReelScreenshot.ts

```diff
@@ -257,3 +261,18 @@ export async function getInstagramPostReelScreenshot(
 		if (page) await closePageSafely({ logger, page });
 	}
 }
+
+export async function handleDialogsInstagram(
+	options: HandleDialogsOptions,
+): Promise<void> {
```

Comment:
Since this is not a common utils, the type should be derived from its parent type.


```suggestion
type HandleInstagramDialogsOptions = Pick<
	NavigateCarouselOptions,
	"logger" | "page"
>;

async function handleInstagramDialogs(
	options: HandleInstagramDialogsOptions,
): Promise<void> {
```